bridge/slack/handlers.go

```diff
@@ -284,11 +289,10 @@ func (b *Bslack) handleDownloadFile(rmsg *config.Message, file *slack.File) erro
 	return nil
 }
 
-func (b *Bslack) fileIsAvailable(file *slack.File) bool {
-	// Only download a file if it is not in the cache or if it has been entered more than a minute ago.
-	if ts, ok := b.cache.Get("file" + file.ID); ok && time.Since(ts.(time.Time)) > time.Minute {
+func (b *Bslack) fileWasRecentlyCached(file *slack.File) bool {
```

Comment:
I would add a godoc comment here with the method to keep a trace of the rationale behind this logic.
Suggestion:
```go
// fileWasRecentlyCached implements Matterbridge's caching logic for files
// shared via Slack.
// 
// We consider that a file was cached if its ID was added in the last minute or
// it's name was registered in the last 10 seconds. This ensures that an
// identically named file but with different content will be uploaded correctly
// (the assumption is that such name collisions will not occur within the given
// timeframes).
```